CorsixTH/Lua/game_ui.lua

```diff
@@ -475,6 +475,10 @@ function GameUI:onCursorWorldPositionChange()
   local room
   if not overwindow and wx > 0 and wy > 0 and wx < self.app.map.width and wy < self.app.map.height then
     room = self.app.world:getRoom(wx, wy)
+  end
+   -- Enable queue icons for patients when hovering the door as well
```

Comment:
Comment slightly off. Your fix is for hovering over the room.

Reply:
My change detects whether the entity you are hovering over has a "room" field attached to it, which doors do. However, I have noticed that only one half of a double-door has that field. So this fix doesn't work perfectly for Operating Theatres or Wards.
I could adjust the check to look for entity.room OR "entity.isMaster = =false", because that would catch the second half of a double door, but that feels like a really janky solution.
I'll keep poking for another, more definitive, way to detect if the entity we are hovering over is a door or not.